src/app/components/NavBar/__snapshots__/index.spec.tsx.snap

```diff
@@ -853,6 +958,13 @@ exports[`content in browser when logged in matches snapshot 1`] = `
         </svg>
       </button>
     </div>
+    <div
+      aria-live="polite"
+      className="c13"
+      id="page-title-confirmation"
+    >
+      Loaded page ""
```

Comment:
does this read out on page load? would we prefer that it be empty at first instead of having "loaded page [silence]"?

Reply:
I have it skip the first non-blank value